apps/src/gamelab/GameLabSprite.js

```diff
@@ -300,3 +300,151 @@ module.exports.AABBops = function(p5Inst, type, target, callback) {
 
   return result;
 };
+
+/**
+ * Returns whether or not this sprite will bounce or collide with another sprite
+ * or group. Modifies the sprite's touching property object.
+ * @method
+ */
+
+module.exports.isTouching = function(p5Inst, target) {
+
+  /*
+   * This code is a subset of the original AABBops method. It needs to be kept in
+   * sync with p5play's method, much like our stateful async version of AABBops
+   * above.
+   */
+
+  var abs = p5.prototype.abs;
+  var round = p5.prototype.round;
+  var quadTree = p5Inst.quadTree;
+  var createVector = p5Inst.createVector;
```

Comment:
Some aren't intentionally, copying what p5play does (but I added a comment for those).
Some should be: fixed in both places in this file (good catch!!)
Some aren't because they aren't called but are used for `instanceof` checks (added a comment)